src/shared/route/useRouteNavigation.ts

```diff
@@ -68,7 +68,7 @@ export const useRouteNavigation = () => {
       body,
     }: {
       companyId: string;
-      body?: PostRouteQuery;
+      body?: PostRouteQuery | null;
     }) => {
       void navigate(CREATE_POST({ companyId }), { state: { ...body } });
```

Comment:
이렇게 바꿔도 예상한대로 동작하지 않나요?

```suggestion
    }) => {
    if (body === null) { // body?: PostRouteQuery를 유지하고 undefined로 확인하는 게 더 깔끔할 것 같기는 함.
      void navigate(CREATE_POST({ companyId }));
      return;
    }
      void navigate(CREATE_POST({ companyId }), { state: { ...body } });
```